### PR TITLE
Bit cmd complete

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

File: pkg/store/string.go [hunk 1]
```diff
@@ -813,12 +909,132 @@ func (s *Store) Strlen(db uint32, args [][]byte) (int64, error) {
...
+			n += int64(bitsInByte[o.Value[pos]])
+		}
 	}
-	return 0, nil
+
+	return n, nil
+}
+
+// BITOP op destkey key [key ...]
+func (s *Store) BitOp(db uint32, args [][]byte) (int64, error) {
```
Comment (siddontang) [inline pkg/store/string.go]: Use BitOP may be better?

File: pkg/store/string.go [hunk 2]
```diff
@@ -768,41 +843,62 @@ func (s *Store) GetRange(db uint32, args [][]byte) ([]byte, error) {
...
 	return nil, nil
 }
 
-func adjustIndex(index int64, min, max int64) int64 {
-	if index >= 0 {
-		return index + min
-	} else {
-		return index + max
+// STRLEN key
+func (s *Store) Strlen(db uint32, args [][]byte) (int64, error) {
```
Comment (siddontang) [inline pkg/store/string.go]: I think this command should be in anther pr, not bit pr here :-)

File: pkg/store/string.go (hunk 1 above)
Comment (qiuyesuifeng) [inline pkg/store/string.go]: i think Op means operation, so maybe use BitOp is ok:)

File: pkg/store/string.go (hunk 2 above)
Comment (qiuyesuifeng) [inline pkg/store/string.go]: ok, got it.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**